plugins/intel-usb4/README.md

```diff
@@ -0,0 +1,32 @@
+# Intel USB4
+
+## Introduction
+
+USB-4 is a hardware interface specification. This plugin supports the Intel Goshen Ridge hardware.
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+an unspecified binary file format, with vendor specific header.
+
+This plugin supports the following protocol ID:
+
+* com.intel.thunderbolt
+
+## GUID Generation
+
+These devices use a custom generation scheme:
+
+* `TBT-00d4b071`
+
+## Update Behavior
+
+All devices will be updated the next time the USB-C plug from the dock is unplugged from the host.
```

Comment:
That is the case for Dell's dock, I don't know it's the case for anyone else's.

Reply:
@bleungatchromium any ideas?